src/nvim/api/buffer.c

```diff
@@ -872,24 +872,25 @@ ArrayOf(Dictionary) nvim_buf_get_keymap(Buffer buffer, String mode, Error *err)
 /// @see |nvim_set_keymap()|
 ///
 /// @param  buffer  Buffer handle, or 0 for current buffer
-void nvim_buf_set_keymap(Buffer buffer, String mode, String lhs, String rhs,
-                         Dictionary opts, Error *err)
+void nvim_buf_set_keymap(uint64_t channel_id, Buffer buffer, String mode,
+                         String lhs, String rhs, Dictionary opts, Error *err)
   FUNC_API_SINCE(6)
 {
-  modify_keymap(buffer, false, mode, lhs, rhs, opts, err);
+  modify_keymap(buffer, false, mode, lhs, rhs, opts, err, channel_id);
```

Comment:
is there a reason for putting the channel id in a different order in this function? might as well keep it consistent otherwise.

Reply:
There's no reason I'll update it .

Reply:
Done